src/usr/local/www/vpn_openvpn_client.php

```diff
@@ -150,6 +151,8 @@
 		} else {
 			$pconfig['shared_key'] = base64_decode($a_client[$id]['shared_key']);
 		}
+		// OpenVPN Defaults to -1 if unset
+		$pconfig['tlsauth_keydir'] = !(empty($a_client[$id]['tlsauth_keydir'])) ? $a_client[$id]['tlsauth_keydir'] : -1;
```

Comment:
`empty()` will return true when the value is `0` which will be a problem since `0` is one of the valid settings for this variable.